app/overrides/hosts_list.rb

```diff
@@ -0,0 +1,13 @@
+Deface::Override.new(
+  virtual_path: 'hosts/_list',
+  name: 'insights_hits_header',
+  insert_before: 'thead tr th.hidden-xs:first-of-type',
+  text: '<th class="hidden-xs" width="10%"><%= _("Insights")%></th>'
+)
+
+Deface::Override.new(
+  virtual_path: 'hosts/_list',
+  name: 'insights_hits_cells',
+  insert_before: 'tbody tr td.hidden-xs:first-of-type',
+  text: '<%= hits_counts_cell(host) %>'
```

Comment:
are we able to pass a react component here?

Reply:
Theoretically yes, but I don't see a reason to do that. The table is completely non-react, so I don't see much benefit from making this specific cell a react component.